templates/README.md

```diff
@@ -1,41 +1,52 @@
 # Commonhaus Templates
 
-We don't want project on-boarding to be a burden.
+We don't want project on-boarding to be a burden, but we have a few requirements to ensure transparency.
 
-The files in this folder are our recommendations for what CF projects should define.
+- `CODE_OF_CONDUCT.md`[^1]: Projects must define a Code of Conduct. They can use the foundation's [Code of Conduct policy][coc_policy], or define their own. See [CODE_OF_CONDUCT.md][COC] and the [Project CODE_OF_CONDUCT.md template][COC_TPL] for examples.
+- `GOVERNANCE.md`[^1]: Projects must explain how decisions are made. See [Governance][GOV] and the [Project GOVERNANCE.md template][GOV_TPL] for examples.
+- `CONTRIBUTING.md`[^1]: Projects must explain how to make contributions. This file must include references to [required contributor agreements](#contributor-agreements). See [CONTRIBUTING.md][CONTRIB] for an example, or [Wrangling Web Contributions: How to Build a CONTRIBUTING.md][mozilla] for other ideas.
 
-- [Code of Conduct](CODE_OF_CONDUCT.md)
-- [Governance](GOVERNANCE.md)
+[^1]: This file can also be defined in the `.github` repository of an organization.
 
-## Other files
+We further recommend:
+
+- **Code Ownership:** Maintain a list of individuals or groups with the authority to review and merge contributions (`CODEOWNERS` or `.github/CODEOWNERS`).
+    - [About Code Owners (GitHub)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
+    - [Code Owners (GitLab)](https://docs.gitlab.com/ee/user/project/codeowners/)
```

Comment:
I would remove this.

If a project thinks a `CODEOWNERS` file is useful, nothing stops the project having it. CF doesn't need to "recommend" it.

```suggestion
```

Reply:
You may not use it with Hibernate, but it remains something the foundation would recommend (as it is conventional, supported by tools, and can make required reviewers more transparent). It is a recommendation, not a requirement, and it is not made idly.

Reply:
How many of the Foundation's current projects use this?

Reply:
JReleaser for one set up CODEOWNERS long time ago, I think to follow the recommendations of the OSSF Scorecard. The project will follow all if nos most of the recommendations laid out by CF once this PR is merged.